packages/sdk-keyimport-utils/index.ts

```diff
@@ -1,28 +1,39 @@
-import { buildKeyImportRequest, SecretKey, SignersInfo, KeyCurve, KeyProtocol } from './codegen/dfns_key_import'
+import {
+  buildKeyImportRequest,
+  SecretKey,
+  SignersInfo,
+  KeyCurve as KeyCurveInternal,
+  KeyProtocol,
+} from '@dfns/dfns-key-import'
 import { ImportWalletBody, GetWalletResponse } from '@dfns/sdk/types/wallets'
 import { ListSignersResponse } from '@dfns/sdk/types/signers'
 
 type Signer = ListSignersResponse['clusters'][number]['signers'][number]
-type KeyScheme = GetWalletResponse['signingKey']['scheme']
+type KeyCurve = GetWalletResponse['signingKey']['curve']
+
+const getCurveAndProtocol = (keyCurve: KeyCurve): { curve: KeyCurveInternal; protocol: KeyProtocol } => {
+  switch (keyCurve) {
+    case 'secp256k1':
```

Comment:
not relevant to this PR, but how are we going to support `Schnorr`? it also uses `secp256k1`.

Reply:
maybe at some point yes 🤷 when we cross that bridge, we’ll need to expose both curve + protocol in the `splitPrivateKeyForSigners()` signature, instead of inferring one as we do now